docker-compose/node-counter.js

```diff
@@ -0,0 +1,4 @@
+var count = 0;
+require('http')
+	.createServer((req, res) => res.end(String(count++)))
```

Comment:
How atomic is this increment?

I also don't like we are introducing another language to this. Could we not use python instead since we are already using it for other things here.

Reply:
Javascript is single-threaded so this is as atomic as it gets. I don't know how to do it in python but I highly doubt that it is as trivial as this.

I agree that new languages should be introduced carefully, but in this case where it's practically a one-liner, I think it's a perfect match for the problem at hand.

Reply:
Ok. I'm still unhappy that we are doing this, but ok. For the record, the equivalent python, while not as trivial, is only slightly less trivial

```python
from http.server import *

count = 0

class Handler(BaseHTTPRequestHandler):
    def do_GET(self):
        global count
        self.send_response(200)
        self.send_header('Content-type','text/html')
        self.end_headers()
        message = str(count)
        count += 1
        self.wfile.write(bytes(message, "utf8"))

with HTTPServer(('0.0.0.0', 3000), Handler) as server:
    server.serve_forever()
```